ndc-client/src/models.rs

```diff
@@ -613,6 +717,27 @@ pub enum MutationOperation {
 }
 // ANCHOR_END: MutationOperation
 
+// ANCHOR: RowUpdate
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(tag = "type", rename_all = "snake_case")]
+#[schemars(title = "Row Update")]
+pub enum RowUpdate {
+    CustomOperator {
```

Comment:
Is there a potential third category here, used for stuff like `inc`? Where we wouldn't need a new value?